UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common.UnitTests/Helper/NMHelperTest.cs

```diff
@@ -0,0 +1,127 @@
+﻿using NUnit.Framework;
+using System.Collections.Generic;
+using UKHO.FileShareClient.Models;
+using UKHO.MaritimeSafetyInformation.Common.Helpers;
+using UKHO.MaritimeSafetyInformation.Common.Models.NoticesToMariners;
+
+namespace UKHO.MaritimeSafetyInformation.Common.UnitTests.Helper
+{
+    public class NMHelperTest
+    {
+        [Test]
+        public void WhenListFilesResponseIsCalled_ThenCheckConversionisProper()
```

Comment:
Could be called.....

WhenNMHelperCallsListFilesResponse_ThenConversionIsCorrect

Reply:
Test Name changed to WhenNMHelperCallsListFilesResponse_ThenConversionIsCorrect